### DISCUSSION TITLE
counter #mahsaamini from twitter API

### DISCUSSION THREAD

**Discussion by ehsanazali** [WeAreMahsaAmini/Core#228] [Ideas]
HEllO,
One idea which we can think of, having a live time #mahsaamini counter from Twitter API, about building and coding I can take the responsibility of it, just write me if you found it useful.
thank you friends.

**Comment by seraheazari** [comment]
[This Chrome extension](https://github.com/seraheazari/MahsaAminiHashflag) will add a HashFlag to the tweets containing [#مهسا_امینی](https://twitter.com/hashtag/%D9%85%D9%87%D8%B3%D8%A7_%D8%A7%D9%85%DB%8C%D9%86%DB%8C?src=hashtag_click), [#MahsaAmini](https://twitter.com/search?q=%23MahsaAmini&src=typed_query), [#OpIran](https://twitter.com/hashtag/OpIran?src=hashtag_click), [#اعتصابات_سراسری](https://twitter.com/hashtag/%D8%A7%D8%B9%D8%AA%D8%B5%D8%A7%D8%A8%D8%A7%D8%AA_%D8%B3%D8%B1%D8%A7%D8%B3%D8%B1%DB%8C?src=hashtag_click) to avoid retweeting manipulated hashtags created by the Iran regime cyber army.

The extension is deployed to chrome extension store and is waiting for review. Will add a link once it's deployed, until then, follow the guide to enable it locally.

**Reply by ehsanazali** [reply]
I don't believe that is quite the same.

**Comment by ghost** [comment]
#no_to_sedition
#نه_به_آشوب
#نه_به_اغتشاش
#صلح
#مرگ_بر_آشوبگر
